core/commonTest/src/kotlinx/serialization/BasicTypesSerializationTest.kt

```diff
@@ -173,6 +173,17 @@ class BasicTypesSerializationTest {
         assertNotSame(umbrellaInstance, other)
     }
 
+    @Test
+    fun testKvSerialization() {
+        if (isWasm()) return
```

Comment:
Please add a comment with YouTrack issue for this wasm bug with float numbers

Reply:
Why did you delete added tests though?

Reply:
I delete only one that was originally ok. There is two similar test with similar testdata. One is passing and the second is not.